src/ui/components/address-action/ApplicationLine/ApplicationLine.tsx

```diff
@@ -93,17 +94,26 @@ function ApplicationImage({
             )}
           />
 
-          <div style={{ position: 'absolute', bottom: 0, right: 0 }}>
+          <div
+            style={{
+              position: 'absolute',
+              bottom: -2,
+              right: -2,
+            }}
+          >
             <NetworkIcon
-              size={16}
+              size={20}
               name={network?.name || null}
               src={network?.icon_url || ''}
-              style={{ borderRadius: 4 }}
+              style={{
+                borderRadius: 8,
+                border: '2px solid var(--neutral-100)',
```

Comment:
should this be the color of current background?

Reply:
You mean `var(--background)`? We don't want this border to be`#fff when appearance set to "light", otherwise it would look like this
<img width="450" alt="image" src="https://github.com/user-attachments/assets/9400b03e-e74d-41b4-824e-14b0d0427818">

Reply:
I mean `var(--surface-background-color)`

Reply:
ah, sure, updated ✅ I will remember our css variable names eventually